en/micro-integrator/docs/administer-and-observe/logs/masking_sensitive_info_in_logs.md

```diff
@@ -33,6 +33,9 @@ You can specify one or more masking patterns in the `<MI-HOME>/conf/deployment.t
 
 With this configuration, each log line is checked for all the configured patterns. If any match is found, it is masked with ‘*****’.
 
+!!! danger "Using single quotes in TOML configs to avoid parsing escape characters"
+    The TOML parser reads the `deployment.toml` file and parse its value before generating a relevant configuration file. If the strings defined in the TOML file are within double quotes, it parses the string with escape characters and writes the resulting value on the config file. Therefore, you can avoid parsing escape characters by using single quotes. This allows the parser to read the TOML value without parsing and write on the config file as it is.
+
```

Comment:
I think it would be enough to instruct the users to use only single quotes. And then we can update the config we have. @madushadhanushka WDYT?

Reply:
+1 to not expose internal implementation. We don't need internal implementation details here regarding file replacement and etc. We can just say that a single quote read the deployment.toml string values as it is and double quotes read along with escape characters. In the documentation, we can have only single quotes as the default way of defining a string value. We can almost ignore having double-quotes.

Reply:
Anyway, I feel we need somewhere a note(This note is fine with me) with the difference between the single quotes and double-quotes. But we don't need to explain internal implementation details.

Reply:
_If the strings defined in the `deployment.toml` file are within double quotes, the string is parsed along with the escape characters. To avoid this, use single quotes when you need to add escape characters as shown in the above example._

Is this better? WDYT?